blockmanager.go

```diff
@@ -1981,55 +1917,43 @@ func (b *blockManager) handleNotifyMsg(notification *blockchain.Notification) {
 		// pool which depends on the transaction is still valid.
 		handleDisconnectedBlockTxns := func(txns []*dcrutil.Tx) {
 			for _, tx := range txns {
-				_, err := txMemPool.MaybeAcceptTransaction(tx, false, true)
+				_, err := b.cfg.TxMemPool.MaybeAcceptTransaction(tx, false, true)
```

Comment:
The point of creating the local `txMemPool` variable is to avoid needing to change all these.